docs/pages/access-controls/guides/device-trust.mdx

```diff
@@ -192,8 +242,105 @@ Device "(=devicetrust.asset_tag=)"/macOS enrolled
 $ tsh logout; tsh login --proxy=example.com --user=alice # fetch new certificates
 ```
 
+<Admonition type="warning" title="Windows">
+On Windows, the `tsh device enroll` command will attempt to elevate to
+administrator privileges on the device. This is required in order to perform a
+TPM Credential Activation. This is only required on enrollment and will not be
+required on future device authentications.
+</Admonition>
+
 The device above is now a trusted device.
 
+## Recommended: Configuring a TPM EKCert CA allow-list
+
+This advice only applies to Device Trust on platforms that use TPMs. For now,
+this is just Windows.
+
+Some TPMs include a certificate (known as an EKCert) signed by the
+manufacturer's CA (Certificate Authority). This certificate allows a third party
+(such a Teleport cluster) to know that the TPM it is communicating with is
+legitimate. This significantly reduces the burden on the administrator to ensure
+that the device has not been tampered with prior to enrollment.
+
+By default, the Teleport cluster does not verify the EKCert. This is because
+not all TPMs include an EKCert, and it is not possible to verify an EKCert
+without knowledge of the manufacturer's CA. This verification is enabled by the
+inclusion of the Teleport configuration field called `ekcert_allowed_cas`.
+
+Once configured, only devices that include a TPM with an EKCert signed by a
+CA specified in the field will be able to enroll. Previously enrolled devices
+will not be affected.
+
+To configure `ekcert_allowed_cas`, you must first obtain the CA certificate in
+PEM format from the manufacturer of the TPM included in your devices. This step
+varies from manufacturer to manufacturer.
+
+Once you have obtained the CA certificate in PEM format, there are two ways of
+configuring `ekcert_allowed_cas`:
+
+- Statically using the Teleport configuration file. This is the simplest
+  option, but is not possible for Teleport Cloud clusters and not recommended
+  for clusters in a highly available configuration
+- Dynamically using `cluster_auth_preference` resource. This works with all
+  clusters and is the most flexible
+
+<Tabs dropDownCaption="Teleport Deployment">
+<TabItem label="Dynamic Resources" options="Self-Hosted,Teleport Enterprise Cloud" >
+Create a `cap.yaml` file or get the existing configuration using
```

Comment:
Suggestion: Since this is necessarily v13+, I would cut all the blurb and go straight for `tctl edit`.

Reply:
Agreed

Reply:
420fd9a1af